runtime/databricks/automl_runtime/errors.py

```diff
@@ -0,0 +1,15 @@
+class AutoMLRuntimeError(Exception):
+    """
+    Base exception class for AutoML Runtime errors.
+    """
+
+    def __init__(self, message="", *args):
+        self.message = message
+        super().__init__(message, *args)
+
+
+class InvalidArgumentError(AutoMLRuntimeError):
```

Comment:
If we don't have lots special logic for the exception, can we just raise `ValueError`? https://stackoverflow.com/a/256260/4195568

Reply:
Yes that makes sense!